ci-operator/step-registry/hypershift/dump/hypershift-dump-chain.yaml

```diff
@@ -5,6 +5,12 @@ chain:
     cli: latest
     commands: |-
```

Comment:
Move these scripts out of the chain definitions and into their own steps in the registry.  Then you'll be able to edit them sensibly in their own `*.sh` files and they'll be verified by our `shellcheck` job.